packages/tpetra/core/src/Tpetra_Details_crsUtils.hpp

```diff
@@ -0,0 +1,401 @@
+// @HEADER
```

Comment:
Ugh, in a previous commit, this file showed up as a rename of `Tpetra_Details_padCrsArrays` (which it is).  But, this latest commit added a lot of comments, perhaps fooling git in to thinking it is not a rename but a new file?  At any rate, the diff is bigger than it need to be because much of the code is from the old file.